thirdparty/blake3/blake3_impl.h

```diff
@@ -46,16 +46,15 @@ enum blake3_flags {
 #if defined(_MSC_VER)
 #include <intrin.h>
 #endif
-#include <immintrin.h>
```

Comment:
Don't we need this anymore?

Reply:
this header seems contains functions for AVX and AVX2 instructions, which is not used by other c file without avx/avx2 postfix.